cli/print.go

```diff
@@ -55,7 +55,7 @@ func printInfo(data ...interface{}) {
 	} else {
 		data = append(data, "\n")
 	}
-	console.Info(data...)
+	console.Infoln(data...)
```

Comment:
What is this solving? I am pretty sure it will break output. We should already add a `\n` above if we aren't printing to console.

Reply:
The line above adds a newline on non-console.

For console output, we want to overwrite the line to keep output clean.

Reply:
oh weird but why does `Cleanup Done.` never adds a new line @klauspost ?